From 3cfd54ec6726e61096c6f37430d7fe75323db9e3 Mon Sep 17 00:00:00 2001
From: Benjamin Otte <otte@redhat.com>
Date: Fri, 3 Oct 2014 16:15:10 +0200
Subject: [PATCH] docs: Add a man page fro gtk3-demo-application

It is an installed binary, so it needs a man page!
---
 docs/reference/gtk/Makefile.am               |  2 ++
 docs/reference/gtk/gtk-docs.sgml             |  1 +
 docs/reference/gtk/gtk3-demo-application.xml | 44 ++++++++++++++++++++++++++++
 3 files changed, 47 insertions(+)
 create mode 100644 docs/reference/gtk/gtk3-demo-application.xml

diff --git a/docs/reference/gtk/Makefile.am b/docs/reference/gtk/Makefile.am
index 968384c..0e94062 100644
--- a/docs/reference/gtk/Makefile.am
+++ b/docs/reference/gtk/Makefile.am
@@ -152,6 +152,7 @@ content_files =					\
 	windows.sgml				\
 	x11.sgml				\
 	gtk3-demo.xml				\
+	gtk3-demo-application.xml		\
 	gtk3-widget-factory.xml			\
 	gtk-query-immodules-3.0.xml		\
 	gtk-update-icon-cache.xml		\
@@ -338,6 +339,7 @@ man_MANS = 				\
	gtk-update-icon-cache.1		\
 	gtk-launch.1			\
 	gtk3-demo.1			\
+	gtk3-demo-application.1		\
 	gtk3-widget-factory.1		\
 	broadwayd.1
 
diff --git a/docs/reference/gtk/gtk-docs.sgml b/docs/reference/gtk/gtk-docs.sgml
index e3956d6..ad448fb 100644
--- a/docs/reference/gtk/gtk-docs.sgml
+++ b/docs/reference/gtk/gtk-docs.sgml
@@ -395,6 +395,7 @@
   <part>
     <title>GTK+ Tools</title>
     <xi:include href="gtk3-demo.xml" />
+    <xi:include href="gtk3-demo-application.xml" />
     <xi:include href="gtk3-widget-factory.xml" />
     <xi:include href="gtk-query-immodules-3.0.xml" />
     <xi:include href="gtk-update-icon-cache.xml" />
diff --git a/docs/reference/gtk/gtk3-demo-application.xml b/docs/reference/gtk/gtk3-demo-application.xml
new file mode 100644
index 0000000..f7a9694
--- /dev/null
+++ b/docs/reference/gtk/gtk3-demo-application.xml
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+               "http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd" [
+]>
+<refentry id="gtk3-demo-application">
+
+<refentryinfo>
+  <title>gtk3-demo-application</title>
+  <productname>GTK+</productname>
+  <authorgroup>
+    <author>
+      <contrib>Developer</contrib>
+      <firstname>Matthias</firstname>
+      <surname>Clasen</surname>
+    </author>
+  </authorgroup>
+</refentryinfo>
+
+<refmeta>
+  <refentrytitle>gtk3-demo-application</refentrytitle>
+  <manvolnum>1</manvolnum>
+  <refmiscinfo class="manual">User Commands</refmiscinfo>
+</refmeta>
+
+<refnamediv>
+  <refname>gtk3-demo-application</refname>
+  <refpurpose>Demonstrate GtkApplication</refpurpose>
+</refnamediv>
+
+<refsynopsisdiv>
+<cmdsynopsis>
+<command>gtk3-demo-application</command>
+</cmdsynopsis>
+</refsynopsisdiv>
+
+<refsect1><title>Description</title>
+<para>
+<command>gtk3-demo-application</command> is an example application
+used by <command>gtk3-demo</command>. There is no need to call it
+manually.
+</para>
+</refsect1>
+
+</refentry>
-- 
1.9.3

